File: e13-emall-parent/e13-seller-api/src/main/java/org/seller/service/SpecificationService.java
package org.seller.service;

import org.common.api.ShopResult;
import org.emall.entity.Specification;
import org.emall.entity.TbSpecification;

public interface SpecificationService {

		//分页查询
		public ShopResult queryByPager(TbSpecification specification , int page , int size);
		
		//新增
		public void add(Specification specification);
		
		//修改前查询
		public Specification findById(Long id);
		
		//修改品牌
		public void update(Specification specification);
		
		//批量删除
		public  void delete(Long[] ids);
}

File: e13-emall-parent/e13-manager-web/src/main/webapp/js/service/specificationService.js
app.service('specificationService',function($http){
		this.findPage=function(page,size,searchEntity){
			return $http.post('../specification/findByPager?page='+page +'&size='+size, searchEntity);
		}
		this.save = function(entity){
			
			return $http.post('../specification/add',entity);
		}
		this.update = function(entity){
			return $http.post('../specification/update',entity);
			
		}
		this.findById = function(id){
			
			return $http.get('../specification/findById?id='+id);
		}
		this.dele = function(selectIds){
			return $http.get('../specification/delete?ids='+selectIds);
		}
		
})